plugin-server/src/main/services/objectStorage.ts

```diff
@@ -0,0 +1,72 @@
+import { PluginsServerConfig } from '../../types'
+import { status } from '../../utils/status'
+
+const aws = require('aws-sdk')
+
+let S3: typeof aws.S3 | null = null
```

Comment:
I'm usually against globals. Why not make `ObjectStorage` into an actual `class`, and store this as an instance variable? The pojo/yolo-object returned from `connectObjectStorage` is stored on the `hub` anyway, so might as well be a real class.